src/components/menu/menu.svelte

```diff
@@ -254,23 +256,29 @@
     display: revert;
   }
 
-  :global .leo-menu-popup ::slotted(leo-menu-item:hover),
-  :global .leo-menu-popup ::slotted(leo-option:hover),
-  :global .leo-menu-popup > leo-menu-item:hover,
-  :global .leo-menu-popup > leo-option:hover {
+  :global :where(.leo-menu-popup) ::slotted(leo-menu-item:hover),
+  :global :where(.leo-menu-popup) ::slotted(leo-option:hover),
+  :global :where(.leo-menu-popup) > leo-menu-item:hover,
+  :global :where(.leo-menu-popup) > leo-option:hover {
```

Comment:
`:where()` reduces the specificity, which makes it easier for the consumer override styles.